### DISCUSSION TITLE
brotli

### DISCUSSION THREAD

**Discussion by Catospye** [tiredofit/docker-nginx-php-fpm#19] [Q&A] [answered]
How can I disable "NGINX_BROTLI_VERSION" as of now I have tried:
(ENV)`NGINX_BROTLI_VERSION` = `FALSE`
and my PHP app still wont load.

**Comment by tiredofit** [comment, marked as answer]
That environment variable is just used during build and for reference as to what commit was used when building Nginx with the Brotli extensions.

What you would want to do is use `NGINX_ENABLE_COMPRESSION_BROTLI=FALSE`.

More details on the parent image (nginx) here: https://github.com/tiredofit/docker-nginx

**Reply by Catospye** [reply]
I ended up doing `NULL` and that worked
